lib/Form/Type/Content/ContentEditType.php

```diff
@@ -19,26 +22,26 @@
  */
 class ContentEditType extends AbstractType
 {
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
This might be a BC break. 
Looking at [Symfony 2.8 upgrade notes](https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#form) I could replace this with:
```
public function getName()
{
    return $this->getBlockPrefix();
}
```
to keep BC, but `getName` should rather be removed (according to notes mentioned above) or return fully-qualified class name [which is already done by `AbstractType`](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/AbstractType.php#L65-L72).
Ping @andrerom @glye